packages/zmarkdown/utils/vfile-assembler.js

```diff
@@ -0,0 +1,53 @@
+const listConcatUnique = (a, b) => {
+  if (typeof a !== 'undefined' || typeof b !== 'undefined') {
```

Comment:
You probably meant `&&`. There's also a simpler way of implementing this if it does what I think it does, something like (not tested):

```
const listConcatUnique = (a, b) => {
if (Array.isArray(a) && Array.isArray(b)) {
  return Array.from(new Set(a.concat(b)))
}
return []
}
```

Reply:
Thanks, fixed in the new version, and your code worked without a change :+1: